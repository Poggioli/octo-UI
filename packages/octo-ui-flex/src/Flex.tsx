import { StyledFlex } from "./styles";

const Flex = StyledFlex;
Flex.toString = () => `.${StyledFlex.className}`;

export { StyledFlex };
export { Flex };
