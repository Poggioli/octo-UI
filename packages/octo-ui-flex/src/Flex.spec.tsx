import { render } from "@testing-library/react";
import { Flex } from "./Flex";

describe("<Flex />", () => {
  it(`GIVEN a <Flex />
      WHEN render is called
      THEN should render a <Flex />`, () => {
    const rendered = render(<Flex />);
    expect(rendered.container).toMatchSnapshot();
  });

  it(`GIVEN a <Flex />
      WHEN toString is called
      THEN return a string`, () => {
    expect(Flex.toString()).toBe(".octo-UI-c-gqwkJN");
  });
});
