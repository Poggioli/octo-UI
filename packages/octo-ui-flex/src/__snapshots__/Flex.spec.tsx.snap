// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Flex /> GIVEN a <Flex />
      WHEN render is called
      THEN should render a <Flex /> 1`] = `
<div>
  <div
    class="octo-UI-c-gqwkJN octo-UI-c-gqwkJN-djZMnX-flexDirection-initial octo-UI-c-gqwkJN-eJbcS-flexWrap-initial octo-UI-c-gqwkJN-fVplCG-justifyContent-initial octo-UI-c-gqwkJN-ebbZJG-alignItems-initial octo-UI-c-gqwkJN-dYGMXV-alignContent-initial octo-UI-c-gqwkJN-bexZoo-alignSelf-initial"
  />
</div>
`;
